core/src/main/clojure/xtdb/query_ra.clj

```diff
@@ -0,0 +1,67 @@
+(ns xtdb.query-ra
```

Comment:
@jarohen Not particular tied to `xtdb.query-ra`. Could also move it to `xtdb.operator` which sits currently empty.